stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -351,10 +354,12 @@ module SwiftGlibc [system] {
         header "${GLIBC_ARCH_INCLUDE_PATH}/sys/select.h"
         export *
       }
+% if CMAKE_SDK != "FREEBSD":
       module sendfile {
         header "${GLIBC_ARCH_INCLUDE_PATH}/sys/sendfile.h"
         export *
       }
+% end
```

Comment:
Changes to this file LGTM, feel free to submit as a separate PR if you would like them to be merged immediately.